src/blogtoblog/BlogToBlogImporter.ts

```diff
@@ -309,6 +313,18 @@ export default class BlogToBlogImporter extends PageImporter {
     });
   }
 
+  cleanName(name: string): string {
```

Comment:
I am not native, but `cleanName` sounds weird to me ;)

```suggestion
  cleanupName(name: string): string {
```